### ISSUE TITLE
Value of type 'NSToolbarItem' has no member 'possibleLabels'

### ISSUE THREAD

**Issue by Gargo** [marioaguzman/toolbar#3]
Can't run example:
```
if  #available(macOS 13.0, *) {
                toolbarItem.possibleLabels = ["Show", "Hide"]
            }
```

**Comment by marioaguzman** [comment]
If you're on Xcode 14 RC or Xcode 14, it does not have the macOS 13 Ventura SDK as per the release notes.

Here's the documentation: https://developer.apple.com/documentation/appkit/nstoolbaritem/3969243-possiblelabels?changes=_3_2&language=objc

**Comment by Gargo** [comment]
@marioaguzman it seems `if  #available(macOS 13.0, *)` doesn't work because I run it on 12.5.1 and inner code should be skipped
